### PR TITLE
C++ 14 Deprecated Tag

### PR DESCRIPTION
**Pull request by bearbin**
Fixes error in https://builds.cuberite.org/job/Cuberite%20Linux%20x64%20Master/384/console

### CODE REVIEW

**Comment by bearbin** [comment]
Should this be merged or is there something broken with my ccache implementation?

**Comment by LogicParrot** [comment]
Appveyor failed. I presume this has nothing to do with ccache.

File: src/Entities/Player.h [hunk 1]
```diff
@@ -218,6 +218,7 @@ class cPlayer :
 
 	/** Forces the player to move in the given direction.
 	@deprecated Use SetSpeed instead. */
+	[[deprecated("Use SetSpeed instead.")]]
```
Comment (LogicParrot) [inline src/Entities/Player.h]: Is this valid c++? o.O

Comment (bearbin) [inline src/Entities/Player.h]: http://josephmansfield.uk/articles/marking-deprecated-c++14.html

Comment (LogicParrot) [inline src/Entities/Player.h]: But I thought we're C++11.

Comment (bearbin) [inline src/Entities/Player.h]: That's what I thought. It's just that I got an error because of the lack of this, and I thought maybe we changed.

**Comment by SphinxC0re** [comment]
We should really consider to use this feature. After all c++14 is two years old :-)

**Comment by LogicParrot** [comment]
Stable > Cutting edge.

**Comment by SphinxC0re** [comment]
c++17 is cutting edge

**Comment by LogicParrot** [comment]
That's just terminology. It's bleeding-edge by my standards :P

The fact is, for portability and stability's sake, we shouldn't rush unless there's a serious incentive.

**Comment by bearbin** [comment]
Thinking over this again, I don't think that this should be merged, instead we should find out what's going wrong with ccache such that it is using C++14 warnings.

**Comment by LogicParrot** [comment]
Note that ccache is supposed to just pass stuff to the compiler, it does not generate warnings internally.

**Comment by tigerw** [comment]
By the way, I'm all for C+14. I think it might just be a flag change.

**Pull request closed without merging**